.changeset/curvy-moons-judge.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/theme': major
```

Comment:
I think this is a minor change according to [semantic versioning](https://semver.org/)

```suggestion
'@shopify/theme': major
```